dotCMS/src/main/java/com/dotcms/business/bytebuddy/WrapInTransactionInterceptor.java

```diff
@@ -0,0 +1,28 @@
+package com.dotcms.business.bytebuddy;
+
+import com.dotcms.business.WrapInTransaction;
+import com.dotmarketing.util.Logger;
+import net.bytebuddy.implementation.bind.annotation.Origin;
+import net.bytebuddy.implementation.bind.annotation.RuntimeType;
+import net.bytebuddy.implementation.bind.annotation.SuperCall;
+
+import java.lang.reflect.Method;
+import java.util.Optional;
+import java.util.concurrent.Callable;
+
+import static com.dotcms.util.AnnotationUtils.getMethodAnnotation;
+
+public class WrapInTransactionInterceptor {
+
+    @RuntimeType
+    // Better to rewrite as an Advice
+    public static Object intercept(@SuperCall Callable<?> delegate, @Origin Method method) throws Exception {
```

Comment:
Oh I see the externalize is here, nvm previous msg

Reply:
No In fact this Intercepter is no longer used and has been replaced as the Advice so my comment to the previous message does apply and this should be removed.   The advice should be more efficient.

Reply:
I have now cleaned up this class